pkg/names/names.go

```diff
@@ -17,6 +17,7 @@ const PROMETHEUS_LABEL_KEY = "prometheus.cnao.io"
 const PROMETHEUS_LABEL_VALUE = "true"
 
 // Relationship labels
+
 const COMPONENT_LABEL_KEY = "app.kubernetes.io/component"
 const PART_OF_LABEL_KEY = "app.kubernetes.io/part-of"
 const VERSION_LABEL_KEY = "app.kubernetes.io/version"
```

Comment:
According golang convention, we should use camelCase rather than upper snake case. e.g.
```suggestion
const VersionLabelKey = "app.kubernetes.io/version"
```

Reply:
Yes, I plan this for another PR :)